controls/cis_rhel8.yml

```diff
@@ -0,0 +1,2285 @@
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - service_avahi-daemon_disabled
+
+  - id: 2.2.5
+    title: Ensure SNMP Server is not enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - service_snmpd_disabled
+
+  - id: 2.2.6
+    title: Ensure HTTP Proxy Server is not enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - package_squid_removed
+
+  - id: 2.2.7
+    title: Ensure Samba is not enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - service_smb_disabled
+
+  - id: 2.2.8
+    title: Ensure IMAP and POP3 server is not enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - service_dovecot_disabled
+
+  - id: 2.2.9
+    title: Ensure HTTP server is not enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - service_httpd_disabled
+
+  - id: 2.2.10
+    title: Ensure FTP Server is not enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - service_vsftpd_disabled
+
+  - id: 2.2.11
+    title: Ensure DNS Server is not enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - service_named_disabled
+
+  - id: 2.2.12
+    title: Ensure NFS is not enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - service_nfs_disabled
+
+  - id: 2.2.13
+    title: Ensure RPC is not enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - service_rpcbind_disabled
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5231
+  - id: 2.2.14
+    title: Ensure RPC is not enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 2.2.15
+    title: Ensure DHCP Server is not enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - service_dhcpd_disabled
+
+  - id: 2.2.16
+    title: Ensure CUPS is not enabled (Automated)
+    levels:
+      - l1_server
+      - l2_workstation
+    automated: yes
+    rules:
+      - service_cups_disabled
+
+  # NEEDS RULE
+  # https://github.com/ComplianceAsCode/content/issues/5232
+  - id: 2.2.17
+    title: Ensure NIS Server is not enabled (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: no
+
+  - id: 2.2.18
+    title: Ensure mail transfer agent is configured for local-only mode (Automated)
+    levels:
+      - l1_server
+      - l1_workstation
+    automated: yes
+    rules:
+      - postfix_network_listening_disabled
```

Comment:
I would also add this variable:
var_postfix_inet_interfaces=loopback-only

Reply:
Good catch - added.